.gitignore

```diff
@@ -131,3 +131,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# Vscode
+.vscode
```

Comment:
since this is specific to your IDE and not a product of the "build" process and everyone is using a different IDE it's better to add that as a configuration in your git global configuration for the machine and not to commit it to every repo you are going to work on.